php-nginx/etc/confd/templates/nginx/site.conf.tmpl

```diff
@@ -21,6 +21,9 @@ server {
     set $access_log_enabled 1;
 
     include /etc/nginx/sites-available/default-*.conf;
+    {{ if ne "" (getenv "WEB_ADDITIONAL_INCLUDES_NAME") }}
+        include /etc/nginx/sites-available/{{ getenv "WEB_ADDITIONAL_INCLUDES_NAME" }}-*.conf;
+    {{ end }}
```

Comment:
To be able to use default-* and another thing where the other thing varies when multiple vhosts are involved.